python-sdk/nuscenes/eval/predict/README.md

```diff
@@ -29,10 +29,8 @@ Note that the evaluation server can still be used to benchmark your results afte
 ## Submission rules
 ### Prediction-specific rules
 * The user can submit up to 25 proposed future trajectories, called `modes`, for each agent along with a probability the agent follows that proposal. Our metrics (explained below) will measure how well this proposed set of trajectories matches the ground truth.
-* Unlike previous challenges, the user will not submit their submissions to the eval server. Instead, we will run the user's model
-on the test set for them. This is because we do not want to interfere with the [nuScenes detection challenge](https://www.nuscenes.org/object-detection?externalData=all&mapData=all&modalities=Any) by exposing the test set annotations. In order for us to run their model, the user should modify the `do_inference_for_submission` function in the [inference script](https://github.com/nutonomy/nuscenes-devkit/blob/nuscenes-predict-challenge/python-sdk/nuscenes/eval/predict/do_inference.py)
-so that their model is used.
 * Up to two seconds of past history can be used to predict the future trajectory for each agent.
+* Unlike previous challenges, the leaderboard will be ranked according to performance on the nuScenes val set. This is because we cannot release the annotations on the test set, so users would not be able to run their models on the test set and then submit their predictions to the server. To prevent overfitting on the val set, the top 5 submissions on the leaderboard will be asked to send us their code and we will run their model on the test set. The winners will be chosen based on their performance on the test set, not the val set.
```

Comment:
"we cannot release the annotations on the test set" -> explain in parentheses that doing so would undercut nuScenes object prediction challenges we run.